readme.md

```diff
@@ -21,8 +21,122 @@ text including copyright information.
 Please visit https://www.teeworlds.com/ for up-to-date information about
 the game, including new versions, custom maps and much more.
 
-Instructions to build the game can be found at 
-https://teeworlds.com/?page=docs&wiki=compiling_everything. In
-particular, you will need SDL2 and FreeType installed.
-
 Originally written by Magnus Auvinen.
+
+
+Building on Linux or macOS
+==========================
+
+Installing dependencies
+-----------------------
+
+    # Debian/Ubuntu
+    sudo apt install build-essential cmake git libfreetype6-dev libsdl2-dev libpnglite-dev libwavpack-dev python3
+
+    # Fedora
+    sudo dnf install @development-tools cmake gcc-c++ git freetype-devel mesa-libGLU-devel pnglite-devel python3 SDL2-devel wavpack-devel
+
+    # Arch Linux (doesn't have pnglite in its repositories)
+    sudo pacman -S --needed base-devel cmake freetype2 git glu python sdl2 wavpack
+
+    # macOS
+    sudo brew install cmake freetype sdl2
+
+
+Downloading repository
+----------------------
+
+    git clone https://github.com/teeworlds/teeworlds --recurse-submodules
+    cd teeworlds
+
+    # If you already cloned the repository before, use:
+    # git submodule update --init
+
+
+Building
+--------
+
+    mkdir -p build
+    cd build
+    cmake ..
+    make
+
+On subsequent builds, you only have to repeat the `make` step.
+
+You can then run the client with `./teeworlds` and the server with
+`./teeworlds_srv`.

```

Comment:
Don't you need to go to the appropriate folder first, @heinrich5991 ?

Reply:
If you followed the above instructions, you already are (they contain `cd build`).

Reply:
Ah, it doesn't work like bam with e.g. `build/x86/debug`